week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -26,12 +24,13 @@ function mood() {
 
 function greaterThan10() {
   const num = 10;
-  const isBigEnough;
-
-  if (isBigEnough) {
-    return "num is greater than or equal to 10";
-  } else {
-    return "num is not big enough";
+  if (num >= 10) {
```

Comment:
there is one issue here, you will never be able to hit "num is not big enough"
fixing it, by setting the condition around isBigEnough only
``` if (num >= 10) {isBigEnough = true;}```